src/prometheus/prometheus-service.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```

Comment:
Isn't this file redundant? The info is already in README file.

Reply:
deleted